main.pm

```diff
@@ -480,6 +480,9 @@ sub load_zdup_tests() {
 
 sub load_consoletests() {
     if (consolestep_is_applicable) {
+        if (check_var("ADDONS", "rt,ha,geo")) {
```

Comment:
shouldn't this be 'if ADDONS *includes* rt'? or does the kmp modules test ONLY make sense if using RT with HA and GEO?